src/styles/blocks/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  background-color: $white-color;
+  max-width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  cursor: pointer;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: $font-size-s;
+    line-height: $line-height-l;
+    font-weight: 500;
+    margin-bottom: 4px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    color: #616070;
+    line-height: $line-height-s;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review-quantity {
+    line-height: $line-height-s;
+  }
+
+  &__price-info {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: $font-size-s;
+    line-height: $line-height-l;
+  }
+
+  &__price-sum {
+    font-size: $font-size-l;
+    line-height: $line-height-l;
+    font-weight: 700;
+  }
+
+  &__purchase-button {
+    display: block;
+    font-size: $font-size-m;
+    line-height: $line-height-m;
+    font-weight: 700;
+    text-decoration: none;
+    text-transform: uppercase;
+    text-align: center;
+    color: $white-color;
+    background-color: $turquoise-color;
+    padding: 11px 0;
+    border: 1px solid $turquoise-color;
+    border-radius: 5px;
+    transition: ease 0.3s;
+
```

Comment:
You can use one empty line between different blocks of style (like style for text, style for position, etc.), then reading your code will be easier

Reply:
Thank you, will change it :)